app.js

```diff
@@ -1,9 +1,9 @@
+require("dotenv").config();
```

Comment:
@deepakr123 - Check and implement envVariables.js

Reply:
resolved